filament/src/Renderer.cpp

```diff
@@ -252,17 +252,32 @@ void FRenderer::mirrorFrame(FSwapChain* dstSwapChain, Viewport const& dstViewpor
     // destination.
     driver.makeCurrent(dstSwapChain->getHwHandle(), mSwapChain->getHwHandle());
 
+    RenderPassParams params = {};
+    // Clear color to black if the CLEAR flag is set.
+    if ((flags & CLEAR) != 0) {
+        params.clear = TargetBufferFlags::COLOR;
+        params.clearColor = {0.f, 0.f, 0.f, 1.f};
+        params.left = 0;
+        params.bottom = 0;
+        driver.getMaxViewportDimensions(&params.width, &params.height);
```

Comment:
I think you could just use `std::numeric_limits<int32_t>::max()` instead of adding a new Driver method.

Reply:
done.  width/height are uint32_t though so I did std::numeric_limits<uint32_t>::max()